Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -202,6 +202,9 @@ public override void DestroyView()
 
         public override void RedrawUI(ViewState viewState)
         {
+            // Store the current focus before Clear/Rebuild
+            Focusable focusedElement = m_ActionsTreeView.focusController.focusedElement;
```

Comment:
this is not a good workaround because the focused element that is retrieved here might not be valid after the m_ActionsTreeView.Rebuild().
The Rebuild can create and/or recycle visualelement. https://docs.unity3d.com/6000.0/Documentation/ScriptReference/UIElements.BaseVerticalCollectionView.Rebuild.html
The focused element might also not represent the right sub visualelement since it's a stack of focused element.

RefreshItems() could mitigate the refres problem but the behavior is not predictable across version
https://docs.unity3d.com/6000.0/Documentation/ScriptReference/UIElements.BaseVerticalCollectionView.RefreshItems.html

Reply:
RefreshItems() doesn't work because the focus is lost through virtualizationController.Refresh() which both call